spring-scaffold-cli/src/main/java/br/com/netodevel/command/template/TemplateHandler.java

```diff
@@ -3,10 +3,7 @@
 import br.com.generate.helpers.ScaffoldInfoHelper;
 import br.com.generator.core.GeneratorOptions;
 import br.com.templates_java.ComposeTemplate;
-import br.com.templates_java.config.jms_aws_sqs.EntryPointMessageGenerator;
-import br.com.templates_java.config.jms_aws_sqs.MessageListenerGenerator;
-import br.com.templates_java.config.jms_aws_sqs.SQSDependencyGenerator;
-import br.com.templates_java.config.jms_aws_sqs.SQSPropertiesGenerator;
+import br.com.templates_java.config.jms_aws_sqs.*;
```

Comment:
The wildcard import `br.com.templates_java.config.jms_aws_sqs.*` can lead to naming conflicts, consider importing only the necessary classes.